docker/stempy-conda-jammy/conda/environment_before.yml

```diff
@@ -0,0 +1,8 @@
+channels:
+  - conda-forge
+dependencies:
+  - python=3.9
+  - pip
+  - numpy=1.23
+  - pip:
+    - cmake
```

Comment:
What are the "before" and "after" environments for?

Reply:
I wanted to make cmake available, but avoid bringing too many libraries into the mix before building stempy. The hdf5 libraries from conda were messing with the build. This may have been fixed by pinning the h5py version, but I did not test it.

Reply:
Are we cool with this?

Reply:
I guess I still don't understand, before and after building stempy?